package.json

```diff
@@ -49,5 +49,8 @@
   "simple-git-hooks": {
     "pre-commit": "npx lint-staged && npm run lint:type",
     "commit-msg": "npx commitlint --edit $1"
+  },
+  "dependencies": {
+    "react-toastify": "9.1.3"
```

Comment:
This package should be added to the frontend folder

Reply:
You can do this by passing workspace to the command - `npm run install <dep name> -w <workspace-name> `